### PR TITLE
[monorepo] Bump packages

### PR DESCRIPTION
**Pull request by Alonski**
### Description

Bumps packages:
 - @counterfactual/apps: 0.1.8 => 0.1.9
 - @counterfactual/cf-adjudicator-contracts: 0.0.3 => 0.0.4
 - @counterfactual/cf-funding-protocol-contracts: 0.0.6 => 0.0.7
 - @counterfactual/cf-wallet.js: 0.2.2 => 0.2.3
 - @counterfactual/cf.js: 0.2.5 => 0.2.6
 - @counterfactual/firebase-client: 0.0.5 => 0.0.6
 - @counterfactual/firebase-server: 0.0.3 => 0.0.4
 - @counterfactual/greenboard: 0.0.2 => 0.0.3 (private)
 - @counterfactual/local-ganache-server: 0.0.4 => 0.0.5
 - @counterfactual/node-provider: 0.2.1 => 0.2.2
 - @counterfactual/node: 0.2.62 => 0.2.63
 - @counterfactual/postgresql-node-connector: 0.0.6 => 0.0.7
 - @counterfactual/simple-hub-server: 0.0.12 => 0.0.13
 - @counterfactual/types: 0.0.37 => 0.0.38
 - @counterfactual/typescript-typings: 0.1.1 => 0.1.2
 - @counterfactual/wallet-ui: 0.1.1 => 0.1.2 (private)

### CODE REVIEW

**Comment by snario** [comment]
Do we really need to bump _everything_?

**Comment by Alonski** [comment]
> Do we really need to bump _everything_?
@snario Since these changes `engines` in all package.json files we need to bump everything...

**Comment by snario** [comment]
Yeah most of them have already been updated post-engine dependency change

**Comment by Alonski** [comment]
This would have prevented #2241

**Comment by cf19drofxots** [comment]
@Alonski what's the updated context of this given the revert we had to do?

**Comment by Alonski** [comment]
@cf19drofxots We need to merge `master` into `next` and then rebase this onto `next`.
Then we need to reapply the bumps.

**Comment by cf19drofxots** [comment]
Got it - @Alonski is this something you're handling or should I pitch in?

**Comment by Alonski** [comment]
@cf19drofxots I can do this tonight. Just want to make sure that we're on board with this.

**Comment by Alonski** [comment]
Ok I'm just gonna bump all packages to make sure

**Comment by Alonski** [comment]
@cf19drofxots Can you check this out?

File: packages/simple-hub-server/package.json [hunk 1]
```diff
@@ -38,10 +38,10 @@
     "url": "https://github.com/counterfactual/monorepo/issues"
   },
   "dependencies": {
-    "@counterfactual/firebase-client": "0.0.5",
-    "@counterfactual/node": "0.2.62",
-    "@counterfactual/types": "0.0.37",
-    "@counterfactual/typescript-typings": "0.1.1",
+    "@counterfactual/firebase-client": "^0.0.6",
```
[the remark this answers is not in the document]
Comment (Alonski) [reply packages/simple-hub-server/package.json]: Must have been caused by a merge conflict :/

[the remark this answers is not in the document]
Comment (Alonski) [reply packages/simple-hub-server/package.json]: Oh actually it seems that `lerna publish` does this by default

[the remark this answers is not in the document]
Comment (Alonski) [reply packages/simple-hub-server/package.json]: BTW @IIIIllllIIIIllllIIIIllllIIIIllllIIIIll and @snario I found a few others that weren't exact pins. Should I just update all of them?

**Pull request merged**